cpp/Platform.Collections.Tests/StackTests.cpp

```diff
@@ -0,0 +1,24 @@
+namespace Platform::Collections::Tests {
+struct EmptyStack : public Stacks::IStack<int> {
+  bool empty() const {}
+
+  void push(int item) {}
+
+  void pop() {}
+
+  int &top() {}
+
+  const int &top() const {}
+};
+TEST(StackTests, Concept) {
+  Stacks::CStack<int> auto stack = std::stack<int>{};
+  stack.push(1);
+  ASSERT_EQ(stack.top(), 1);
+  stack.pop();
+  ASSERT_TRUE(stack.empty());
+}
+TEST(StackTests, Interface) {}
```

Comment:
Why do we need this empty test?

Reply:
To write: `Stacks::CStack<int> auto stack = EmptyStack;`

Reply:
Are you sure? We have this code statement in the next test.